### PR TITLE
login api functionality

### PR DESCRIPTION
**Pull request by kommusrinivasulu**

### CODE REVIEW

File: ClariseMain/src/main/java/com/clairvoyant/clarise/configuration/constants/CommonConstants.java [hunk 1]
```diff
@@ -3,4 +3,9 @@
 public class CommonConstants {
     public static final String AUDIT_TRAIL_EXCEPTION =
         "There are problems in saving the audit details for this operation!!";
+   
+    public static final long JWT_TOKEN_VALIDITY = 2 * 60 * 60;
+    public static final String TOKEN_PREFIX = "Bearer ";
+    public static final String HEADER_STRING = "Authorization";
```
Comment (santoshgaikwad14) [inline ClariseMain/src/main/java/com/clairvoyant/clarise/configuration/constants/CommonConstants.java:9]: Use existing constant from `org.springframework.http.HttpHeaders.AUTHORIZATION` where ever required.

**Review by santoshgaikwad14 — APPROVED** [review]
Approved with minor changes.

File: ClariseMain/src/main/java/com/clairvoyant/clarise/configuration/constants/CommonConstants.java (hunk 1 above)
Comment (kommusrinivasulu) [reply ClariseMain/src/main/java/com/clairvoyant/clarise/configuration/constants/CommonConstants.java]: Resolved

**Review by santoshgaikwad14 — APPROVED** [review]
Approved.

**Pull request merged**